helm/csi-powermax/k8s-1.23-values.yaml

```diff
@@ -1,5 +1,5 @@
 # IT IS RECOMMENDED YOU DO NOT CHANGE THE IMAGES TO BE DOWNLOADED.
-kubeversion: "v1.20"
+kubeversion: "v1.23"
 
```

Comment:
it would be good to assess the sidecars and arrive at a common ground.

Reply:
hi @prablr79  , I checked that no new versions of sidecars available . Right now we are using all the latest .

Reply:
@boyamurthy  please do check there is new sidecar need to update https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v2.4.0

Reply:
Thanks @prablr79  , updated the sidecar .

Reply:
@boyamurthy looks like the changes still not reflected.